### DISCUSSION TITLE
Getting 307 on ALB ingress for argocd v2.2.2

### DISCUSSION THREAD

**Discussion by bessyG** [argoproj/argo-cd#8199] [Q&A] [answered]
I had setup ALB ingress for argocd . And it was working fine on last week . But Friday onwards (when I recreated the ingress) it's getting 307 error in the loadbalancer. Confused. 😐  Please give a solution. I used the following method 
https://argo-cd.readthedocs.io/en/stable/operator-manual/ingress/#aws-application-load-balancers-albs-and-classic-elb-http-mode

**Comment by ishitasequeira** [comment]
@bessyG While following the docs, did you create the Service along with the Ingress or just the Ingress?

**Reply by bessyG** [reply, marked as answer]
Hey I fixed this. yes  service was already there. The fix is to add HTTPS as the health check protocol
```alb.ingress.kubernetes.io/healthcheck-protocol: HTTPS```

Thanks for your reply. 🤝

**Reply by kitt1987** [reply]
If TLS is disabled by the --insecure flag, then the health check protocol should be HTTP. Otherwize, use HTTPS instead.
